electionleaflets/urls.py

```diff
@@ -106,6 +119,11 @@ def get_redirect_url(self, *args, **kwargs):
         HomePageRedirectView.as_view(),
         name="constituency-view",
     ),
+    re_path(
+        r"^leaflets/(?P<pk>[^/]+)/images/$",
```

Comment:
Minor thing, but the pk pattern in the original route was `(?P<pk>\d+)` (i.e: only digits).
Here you're changing it to anything but a slash.
Is that on purpose or just copy/paste from the example above?

Reply:
Good point, it should match the destination to ensure we don't redirect to a 404 :+1: